README.rst

```diff
@@ -6,122 +6,86 @@ Taisei
 Introduction
 ------------
 
-Taisei is an open clone of the Tōhō Project series. Tōhō is a one-man project of
-shoot-em-up games set in an isolated world full of Japanese folklore.
+Taisei Project is an original open source fan-game set in the world of Tōho
+Project. It is a top-down vertical-scrolling curtain fire shooting game (STG),
+also known as "danmaku" or "bullet hell," with an original soundtrack,
+character art, and story.
 
-Installation
-------------
+Taisei Project is written in C, with an SDL2/OpenGL renderer. It is available
+for Windows, Linux (x86_64), and macOS.
```

Comment:
> with an SDL2/OpenGL renderer.

This is a little misleading, because we are not using SDL for rendering. SDL *does* have an optional 2D renderer, but it's too limited for us. SDL is only used as a platform abstraction layer.

> Taisei Project is written in C, 

Perhaps it is worth mentioning that it is specifically written in C11. Unfortunately many compilers still do not support that, including Microsoft's.

> It is available for Windows, Linux (x86_64), and macOS.

Maybe something like "It works on many operating systems, including but not limited to Windows, Linux, and macOS"? Those three have the "official support", but it's known to run on some other systems as well. And it's probably worth mentioning that it can run in the browser too; that's all the rage these days apparently.

Reply:
Done and done.